js/main.js

```diff
@@ -20,4 +19,95 @@ function getRandomNumber(min, max, decimal) {
   return randomNumber;
 }
 
-getRandomNumber(100, 200, 5);
+const TITLE = [
+  'Восхитительный дворец',
+  'Караоке лаунж',
+  'Гостевой домик',
+  'Домик уточки',
+  'Аквадискотека',
+];
+
+const TYPE = [
+  'palace',
+  'flat',
+  'house',
+  'bungalow',
+  'hotel',
+];
+
+const CHECKIN = [
+  '12:00',
+  '13:00',
+  '14:00',
+];
+
+const CHECKOUT = [
+  '12:00',
+  '13:00',
+  '14:00',
+];
+
+const FEATURES = [
```

Comment:
```suggestion
const TITLES = [
  'Восхитительный дворец',
  'Караоке лаунж',
  'Гостевой домик',
  'Домик уточки',
  'Аквадискотека',
];

const TYPES = [
  'palace',
  'flat',
  'house',
  'bungalow',
  'hotel',
];

const CHECKINS = [
  '12:00',
  '13:00',
  '14:00',
];

const CHECKOUTS = [
  '12:00',
  '13:00',
  '14:00',
];

const FEATURES = [
```
Массивы должны быть во множественном числе